datasource/src/test/scala/quasar/physical/mongo/MongoScalarStagesInterpreterSpec.scala

```diff
@@ -66,12 +66,29 @@ class MongoScalarStagesInterpreterSpec
     }
   }
 
+  "Pivot special" >> {
+    "no unnecessary undefined pairs" in {
```

Comment:
I'm in the process of adding more `IncludeId` and `IdOnly` pivot tests in `ScalarStagesSpec`. I'll add this one there, and then we can delete it here. In general, we should put all the tests we can in `ScalarStagesSpec` and only put tests specific to individual implementations here.

Reply:
Added here https://github.com/slamdata/quasar/pull/4101/files#diff-018b1055badf3c3df25b2b218bb1abcbR1161